### ISSUE TITLE
s2anet 训练旋转矩形检测速度很慢

### ISSUE THREAD

**Issue by xbc922** [PaddlePaddle/PaddleDetection#7248]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

数据集约1万张图，4张16G显存v100 batch_size 10 ，lr0.05 一个batch 要差不多3分钟，看了下

[11/02 09:14:21] ppdet.engine INFO: Epoch: [0] [  0/200] learning_rate: 0.016667 loss: 4.026768 fam_cls_loss: 1.191281 fam_reg_loss: 1.182869 odm_cls_loss: 1.165921 odm_reg_loss: 0.486696 eta: 1 day, 22:42:04 batch_cost: 10.5078 data_cost: 1.3552 ips: 0.9517 images/s
[11/02 09:17:41] ppdet.engine INFO: Epoch: [0] [ 20/200] learning_rate: 0.017333 loss: 2.455547 fam_cls_loss: 0.776772 fam_reg_loss: 0.681920 odm_cls_loss: 0.615343 odm_reg_loss: 0.256558 eta: 1 day, 20:27:41 batch_cost: 9.9918 data_cost: 0.0006 ips: 1.0008 images/s

**Comment by xbc922** [comment]
GPU 利用率几乎是一直100%

**Comment by wangxinxin08** [comment]
你的配置文件有做什么修改吗？可以发下看看
